cypress/integration/report_page_test.js

```diff
@@ -0,0 +1,104 @@
+describe("Report Page Test", function(){
+    beforeEach(function(){
+        cy.searchKeyword("GST");
+        cy.get(".result").click();
+        cy.url().should("include","/report");
```

Comment:
The URL will change before the contents change, so I think a better assertion to block Cypress from continuing until the page is ready, is to query for a DOM element that only exists when the page is ready:
`cy.get('.report-page-heading').should("exist");`
Some useful info about this: https://sqa.stackexchange.com/questions/44539/how-to-wait-till-all-components-are-loaded-in-cypress